programs/mango-v4/src/accounts_ix/openbook_v2_liq_force_cancel_orders.rs

```diff
@@ -35,7 +35,7 @@ pub struct OpenbookV2LiqForceCancelOrders<'info> {
     #[account(
         has_one = bids,
         has_one = asks,
-        has_one = event_queue,
+        has_one = event_heap,
```

Comment:
these renames sound to me like they should be in a follow-up pr, not this dependency-adjusting one (but it's ok if it's just this single thing)

Reply:
right, yet I'm not sure if I can get an update openbook version without the rename

Reply:
ah, because of the has_one constraints, got it!